### ISSUE TITLE
Day display order error in v-date-picker

### ISSUE THREAD

**Issue by alanscordoba** [vuetifyjs/vuetify#19070]
### Environment
**Vuetify Version:** 3.4.10
**Vue Version:** 3.4.14
**Browsers:** Chrome 120.0.0.0
**OS:** Windows 10

### Steps to reproduce
Try link example.

### Expected Behavior
May the days appear from Sunday.

### Actual Behavior
The days start with Monday.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpdj02OwjAMha9iZZMZqTQzghUqSNwBsaEsQnAhkKZRahCo6t1J01b8rGI/f47f2zas9krcrki6eKTnms2ZLl3lCRpQHiXhpp9BC4WvSuADy3ObW7xHVFW2JhgGsPhc/GlyC0AnLHEOsQY4YCGvhta9yI0+nogn/UyI7m1jZyolzWttbDnWI11IY/ZSXd7EbrX9zS1LYraVc2mwFoJlhKUzwdmyw7LbRDoXy9iEECS1RT9IUTwEeuK0uqAHMbLiG+6U/q9MvN1gbcKm6Sz9nwUrsfhjuyfAl3dO)

**Comment by K-Schaeffer** [comment]
This is not an error, is the intended behavior.
Still, would be nice to have again the prop to set the firstDayOfTheWeek as v2.

Relates to https://github.com/vuetifyjs/vuetify/issues/18844

**Comment by alanscordoba** [comment]
I agree, but the strange thing is that if the local configuration is not used, the first day is Sunday:
`export const vuetify = createVuetify({
  locale: {
    locale: 'es',
    fallback: 'es',
  },
})`
